docs/stencil-docs/page-builder/product-widget-tutorial.md

```diff
@@ -73,65 +57,146 @@ Accept: application/json
   "template": "<div style=\"text-align:center\">\n<h1>{{_.data.site.product.name}}</h1>\n<div>\n<img src=\"{{_.data.site.product.defaultImage.url}}\">\n</div>\n<div>\n<p>${{_.data.site.product.prices.price.value}}</p>\n</div>\n</div>"
 }
 ```
+
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner)
 
+**[Response:](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#responses)**
+
+```json
+{
+  "data": {
+    "uuid": "479e6ffe-73bc-4a0c-8e49-654d97691e8b",
+    "name": "Product Widget",
+    ...
+}
+```
+
+|Property|Type|Description|
+|-|-|-|
+|`name`|string|Name of the widget|
+|`template`|string|The [widget template](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview#widget-templates)|
```

Comment:
```suggestion
|`template`|string|The [widget template](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview#widget-templates) rendered as Handlebars HTML.|
```

Reply:
Since we're linking to the widget template section in the widget overview, do we really need to include "rendered as handlebars HTML"?

Reply:
I found that clarification helpful but I will leave this for Uwana to make the final determination.

Reply:
@ektravel, I agree with @aglensmith that mentioning Handlebars immediately after the link is redundant since the first sentence on the linked page talks about Handlebars. We can remove that portion of the sentence.